sysdig_client_test.go

```diff
@@ -0,0 +1,160 @@
+// +build unit
+
+package sysdig_client_test
+
+import (
+	"encoding/json"
+	sysdigclient "github.com/NeowayLabs/sysdig-client"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func getSumMetricConfiguration() (string, []sysdigclient.Metric, sysdigclient.Period) {
+	filter := "agent.tag.team = \"datapirates\" and bot.name = \"abi\""
+
+	metrics := []sysdigclient.Metric{
+		{
+			Id: "bot.result.status.2xx.total",
+			Aggregations: sysdigclient.Aggregation{
+				Time:  "sum",
+				Group: "sum",
+			},
+		},
+	}
+	period := sysdigclient.Period{}
+
+	return filter, metrics, period
+}
+
+func getSumMetricResponseRecord() map[string]interface{} {
+	return map[string]interface{}{
+		"data": []map[string]interface{}{
+			{
+				"d": []int{
+					80171,
+				},
+			},
+		},
+		"start": 1568127600,
+		"end":   1568214000,
+	}
+}
+
+func TestGetSumMetricWithDaysPeriod(t *testing.T) {
+
+	filter, metrics, period := getSumMetricConfiguration()
+	period.Days = 30
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
this handler are duplicated with handler below.. you could add some setup() function..

Reply:
Adjusted!